package.json

```diff
@@ -1,25 +1,35 @@
 {
   "name": "@mobtimeapp/web",
   "version": "0.0.0-beta.0",
-  "description": "Mobtime web ui",
-  "main": "src/ui.js",
+  "description": "MobTime web front-end built with HyperApp and Vite.",
+  "author": "Alex Barry",
+  "license": "ISC",
+  "type": "module",
   "scripts": {
-    "start": "parcel support/index.html",
-    "tailwind": "tailwindcss -i ./support/tailwind.css -o ./support/index.css",
-    "test": "ava"
+    "tailwind": "tailwindcss build -i ./src/styles/index.css -o ./src/styles/styles.css -c ./tailwind.config.js",
+    "test": "ava",
+    "dev": "vite",
```

Comment:
Might be able to rename this from `dev` to `start`, I think. Makes it easier for the `npm` folk since `npm start` is a thing npm understands, but dev requires `npm run dev`. I'm not dead set one way or another, but I do like less typing 🤷

Reply:
sounds good!